plugins/base/src/main/kotlin/parsers/MarkdownParser.kt

```diff
@@ -567,8 +568,20 @@ open class MarkdownParser(
         }
 
         //Horrible hack but since link resolution is passed as a function i am not able to resolve them otherwise
+        @kotlin.Suppress("DeprecatedCallableAddReplaceWith")
+        @Deprecated("This function makes wrong assumptions and is missing a lot of corner cases related to generics, " +
+                "parameters and static members. This is not supposed to be public API and will not be supported in the future")
         fun DRI.fqName(): String? = "$packageName.$classNames".takeIf { packageName != null && classNames != null }
```

Comment:
The overall problem is basically here.

For `com.example.Object.function()` it would return `com.example.Object`, so if you have multiple `@see` tags that point to multiple different functions from the same `object` - there would be name collisions, and some members would be discarded by `groupBy` further down below.